awx/main/tests/functional/test_inventory_source_injectors.py

```diff
@@ -192,10 +193,15 @@ def create_reference_data(source_dir, env, content):
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('this_kind', CLOUD_PROVIDERS)
-def test_inventory_update_injected_content(this_kind, inventory, fake_credential_factory, mock_me):
+def test_inventory_update_injected_content(inventory, fake_credential_factory, self):
```

Comment:
The trailing args here need to be fixed to keep the original fixture (maybe) and drop `self` that doesn't exist.

Reply:
This has been reverted here: https://github.com/ansible/awx/pull/15537/commits/1a3df541b2f38746c5415c3c82fd37c755b598c8